bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcParametersProvider.scala

```diff
@@ -46,10 +47,16 @@ object ItcParametersProvider {
       itcPanel.conditions.right
 
     def spatialProfile: String \/ SpatialProfile =
-      itcPanel.spatialProfile.fold("Spatial profile not available".left[SpatialProfile])(_.right)
+      for {
+        tEnv <- targetEnvironment
+        sp   <- tEnv.getBase.getTarget.getSpatialProfile.fold("Spatial profile not available".left[SpatialProfile])(_.right)
```

Comment:
Target environment is immutable and throws an exception if you try to create it with a `null` target so it should be fine to assume it is set to some value.